Runtime/AvatarCreator/WebRequests/AvatarApi.cs

```diff
@@ -103,7 +93,7 @@ public async Task<ColorPalette[]> GetAllAvatarColors(string avatarId)
             var response = await authorizedRequest.SendRequest<Response>(
                 new RequestData
                 {
-                    Url = AvatarEndpoints.GetColorEndpoint(avatarId),
+                    Url = $"{Endpoints.API_V2_BASE_URL}avatars/{avatarId}/colors?type=skin,beard,hair,eyebrow",
```

Comment:
This is the crux of point 3, and if used everywhere will negate the need for the endpoints classes which we can then remove.

Reply:
While I do agree it is kinda weird that we have an endpoint class, I don't really know of a better way to do it unless we can isolate their usage into their own classes or something (no need to share across scripts). In general I feel like the endpoints are all over the place so it's hard to group them together in a way that makes sense.

Reply:
Also I like using constants where possible. If you ever need to update something EG this lot of colors params, perhaps we add a new category, I think it's more difficult to find if it is hidden own inside a function rather than up the top in a constant. But perhaps that is just personal preference. 😅

Reply:
How do you mean? This proposed change would remove the need for them, right (other than the base one?). Where is the endpoint declaration needed other than the methods which make the API requests?

IMO the base one should become a environment file (like a .env, or an appsettings.json in .net APIs) and would then be more common place. I also have a cool idea how to implement that in a way to make it easy to switch between dev env and prod env.